docs/source/markdown/podman-build.1.md.in

```diff
@@ -812,6 +812,10 @@ types include:
 
 Unset environment variables from the final image.
 
+#### **--unsetlabel**=*label*
+
+Unset the image label, causing the label not to be inherited from the base image.
+
```

Comment:
I see no wiring into the podman options!

Reply:
@Luap99 for regular podman it should be automatically wired via buildah for `-remote` i just added a commit.

Reply:
The flag is a NOP right now, look in cmd/podman/common/build.go you have to assign it to the BuildOptions struct in order to get the actual value.

Reply:
ah fixed it.